src/components/Atoms/Text/Text.md

```diff
@@ -1,46 +1,86 @@
 ```js
 <div>
+  <Text
+    tag="h1"
+    family="Anton"
+    uppercase
+    weight="normal"
+    size="super"
+    mobileColor="blue"
+  >
+    <b style={{ width: "100px", display: "inline-block" }}>H1 </b>
+    <Text style={{ textTransform: "none" }} tag="span" size="s">
+      <b>family:</b> Anton <b>weight:</b> Regular <b>size:</b> 80px / 5rem /
```

Comment:
Let's add some copy here to these ace new examples to clearly state what's happening:

`<b>family:</b> Anton <b>weight:</b> Regular <b>size:</b> 80px / 5rem / mobileColour: green` etc.